<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-GCUS0J0" failures="0" tests="1" name="Default test" time="0.049" errors="0" timestamp="2024-11-19T17:09:20 IST">
  <testcase classname="io.jans.configapi.BaseTest" name="@BeforeSuite initTestSuite" time="0.036">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:208)
at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:216)
at java.base/java.nio.file.Path.of(Path.java:147)
at java.base/java.nio.file.Paths.get(Paths.java:69)
at io.jans.configapi.BaseTest.initTestSuite(BaseTest.java:122)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:382)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeSuite initTestSuite -->
  <testcase classname="io.jans.configapi.BaseTest" name="@BeforeMethod getAccessToken" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod getAccessToken -->
  <testcase classname="io.jans.configapi.test.auth.ClientResourceTest" name="postClient2" time="0.0">
    <skipped/>
  </testcase> <!-- postClient2 -->
  <testcase classname="io.jans.configapi.test.auth.ClientResourceTest" name="postClient" time="0.0">
    <ignored/>
  </testcase> <!-- postClient -->
  <testcase classname="io.jans.configapi.test.auth.ClientResourceTest" name="getClients" time="0.0">
    <ignored/>
  </testcase> <!-- getClients -->
</testsuite> <!-- Default test -->
